src/bin/pgcopydb/summary.c

```diff
@@ -1866,12 +1866,6 @@ summary_stop_timing(DatabaseCatalog *catalog, TimingSection section)
 			return false;
 		}
 
-		if (!summary_pretty_print_timing(catalog, timing))
-		{
-			/* errors have already been logged */
-			return false;
-		}
-
```

Comment:
How does that work now with respect to the `duration_pretty` column that still exists in the SQLite catalogs?

Reply:
Looking at the existing sources, there seems to be ne need to keep the field in the table.